### PR TITLE
chore: update PR template

### PR DESCRIPTION
**Pull request by lisli1**
#### Describe the change

(Follow-up to team discussion) Modify PR template to remind PR creators to state the motivation for their changes. This will help reviewers understand the context and reasoning behind the PR.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+<!-- Are there any parts that you've intentionally left out-of-scope for a later PR to handle? -->
+
+<!-- Were there any alternative approaches you considered? What tradeoffs did you consider? -->
+
+#### Pull request checklist
+<!-- If a checklist item is not applicable to this change, write "n/a" in the checkbox -->
+- [ ] Addresses an existing issue: #0000
 
 > Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.
```
Comment (chingucoding) [inline .github/pull_request_template.md:19]: Would it make sense to have that be a comment in the PR template given that this mostly is addressed to the PR creator, not the reviewer?

**Review by JGibson2019 — APPROVED** [review]
LGTM! Thanks for updating the templates!

**Review by RobGallo — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (lisli1) [reply .github/pull_request_template.md:19]: Thank you for your review! To be clear, you are referencing line 19 (previously line 5), which reads `Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.`? 

The team agrees that this line isn't really necessary, with checks already configured to run automatically in the PR builds, so I will remove it.

If you were referring to a different line, please feel free to open a PR with your suggestion.

Comment (chingucoding) [reply .github/pull_request_template.md:19]: Yes, I was referring to that line! Thank you for removing that line!

**Review by chingucoding — APPROVED** [review]
Looks good to me.

**Pull request merged**